src/main/flight/pid.h

```diff
@@ -120,6 +120,8 @@ typedef struct pid_s {
 
     float collective;
 
+    float exampleFloat;            // Active value to be used in PID
+
```

Comment:
Usually parameters are not used directly from the config storage variable, but copied to a local "runtime" variable.
This is important specifically with PID parameters.